crates/uv/src/bin/uvx.rs

```diff
@@ -11,7 +11,7 @@ fn run() -> Result<ExitStatus, anyhow::Error> {
         bail!("Could not determine the location of the `uvx` binary")
     };
     let uv = bin.join("uv");
-    let args = ["tool", "run"]
+    let args = ["tool", "x"]
```

Comment:
Maybe... `uvx` instead of `x`?

Reply:
I think `x` just felt a bit random, where `uvx` is clearer RE why it exists and less confusing if someone accidentally runs it.